src/app/(app)/members/components/MemberFormBuilder.tsx

```diff
@@ -87,7 +87,7 @@ export const userBuilderSchema = z.object({
   phoneNumber: z
     .string({ message: "Phone number is required" })
     .regex(PHONENUMBER_VALIDATOR_REGEX, "Phone number is not valid"),
-  date_of_birth: z.date().max(new Date(Date.now()), "under age"),
+  date_of_birth: z.date().max(new Date(Date.now()), "Under age"),
```

Comment:
For members, we only allow people who are older than 16 years. Can we incorporate this validation here?

Reply:
Sure, I fixed the validation here, but we need to customize the date of birth calendar. I've created a separate ticket for this and will start working on it